src/mixins/java/org/spongepowered/common/mixin/api/minecraft/world/level/EntityGetterMixin_API.java

```diff
@@ -52,6 +52,8 @@
     @Shadow List<? extends net.minecraft.world.entity.player.Player> shadow$players();
     //@formatter:on
 
+    @Shadow List<net.minecraft.world.entity.Entity> getEntities(@org.jetbrains.annotations.Nullable net.minecraft.world.entity.Entity param0, net.minecraft.world.phys.AABB param1);
```

Comment:
* This needs to be in the `//formatter` block above
* Use `shadow$getEntities()`